0100-private-projects.md

```diff
@@ -0,0 +1,36 @@
+- Feature Name: Private Projects
+- Created: 2020-02-10
+- Associated Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1602490
+
+# Summary
+
+Add an option to make a project "private", restricting who can see and interact with it.
+
+# Motivation
+
+It can be useful in some cases to be able to hide a project from users. For example, when a superuser creates a new project, they might want to make sure that everything is correctly set up before exposing it — running sync, verifying that it is enabled for the right locales, etc. In such a case, the project being not visible from non-superusers is enough. Once the superuser is done with configuration, they simply make the project public.
+
+For a different use-case, let's consider an agency working with several clients. They have a number of projects created on Pontoon, and they want to have specific localizers working on specific projects. It would be beneficial for them to able to restrict seeing a project to only a defined list of users. This requires being able to mark a project as private, and also having a configuration option to select a list of users who will be able to see, and interact, with that project. That would allow a clear separation of projects, reducing the risk of localizers translating the wrong set of projects.
```

Comment:
Grammar: beneficial for them to **be**  able to.

Reply:
Additionally, what if a translator decides to pre-translate some strings when the project is private? IMHO Such contributions shouldn't be visible in the action log.

Reply:
Latest activity probably needs to be computed differently indeed, to not show private projects activity. 

Blocking translation however defeats the purpose of having this feature for a company like wanadev, as the ultimate goal to have only a selected group of users able to translate.